oaknational/Oak-Web-Application
# Curric/unit modal CUR-267

## Curriculum Modal (MVP)

- Adds Sidebar component and test file
- Lesson Metadata component and test file
- Adds Unit Modal component and test file
- Adds Unit Info button to Unit Tab component

## Issue(s)

Fixes #CUR-267

## How to test

1. Go to https://deploy-preview-1953--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-secondary-aqa/units
2. Click on unit info buttons to open modal
3. You should see unit info for that specific modal along with the LessonMetadata component
4. Click on a unit with optional units 
5. You should se unit optionality card and coming soon buttons to access those units

## Screenshots

How it used to look (delete if n/a):
<img width="934" alt="Screenshot 2023-09-29 at 18 23 15" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/fd409f5c-0642-4af8-9c9c-6fab681487c9">

How it should now look:
<img width="798" alt="Screenshot 2023-10-02 at 11 56 17" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/62249cc7-0fc2-4c80-b0e2-069ad9dc547d">
<img width="756" alt="Screenshot 2023-10-02 at 11 56 05" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/051f8500-6005-4691-a8bf-5d08edac7dac">
<img width="742" alt="Screenshot 2023-10-02 at 11 55 46" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/13f4c807-5748-431a-b2c2-1fbf3361fb15">



## Checklist
